docker-compose.yml

```diff
@@ -1,27 +1,19 @@
-version: '2'
+version: '3'
 services:
-  postgres:
+
+  stringer-postgres:
```

Comment:
Do these names need to change to account for multiple apps running on the same machine?

Reply:
Technically they don't *need* to change, but yes - if you launch the current docker compose, it'll create two containers nicknamed `postgres` and `web`, but those names aren't very descriptive. In my machine I have 14 different containers running, and one "central" postgres container, so having them named properly / more descriptively is helpful. I think this was also part of someone else's suggestion (maybe @Ni-g-3l ?).

Reply:
Yes,  I agree, I have something like 20 containers on same instance, and many of them are database for service. So I think it's good practice to prefix your services with the app name. It make the instance maintenance process easier.